baker/algolia/utils/pages.ts

```diff
@@ -246,17 +268,14 @@ function generateGdocRecords(
             )
             continue
         }
+
         // Only rendering the blocks - not the page nav, title, byline, etc
-        const renderedPostContent = ReactDOMServer.renderToStaticMarkup(
-            createElement(
-                "div",
-                null,
-                createElement(ArticleBlocks, {
-                    blocks: gdoc.content.body,
-                })
-            )
-        )
-        const chunks = generateChunksFromHtmlText(renderedPostContent)
+        const plaintextContent = new MarkdownTextWrap({
+            text: formatGdocMarkdown(gdoc.markdown || ""),
+            fontSize: 12,
+        }).plaintext
+
+        const chunks = generateChunksFromHtmlText(plaintextContent)
```

Comment:
Now that this is handling plaintext, it can also be somewhat refactored, and we can also get rid of the `html-to-text` dependency in there.